suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -73,6 +74,23 @@ const sectionListContainerStyle = prepareNativeStyle(utils => ({
     paddingVertical: utils.spacings.small,
 }));
 
+const sortKeysPendingFirst = (a: string, b: string) => {
+    if (a === 'pending') return -1;
+    if (b === 'pending') return 1;
+
+    const dateA = new Date(a);
+    const dateB = new Date(b);
+
+    return dateA.getTime() - dateB.getTime();
+};
+
+const sortPendingTransactions = (a: WalletAccountTransaction, b: WalletAccountTransaction) => {
+    if (a.blockTime === undefined) return -1;
```

Comment:
what if if both are undefined? I think there should be also 

`if (a.blockTime === undefined && b.blockTime === undefined) return 0` as first condition

Reply:
52579935427dce97d9ef1257f571407e46f5bdbc